build.cmd

```diff
@@ -153,18 +153,42 @@ if /i '%ARG%' == 'ci_part1' (
     set TEST_NET40_COREUNIT=1
     set TEST_PORTABLE_COREUNIT=1
     set TEST_CORECLR=0
-    set TEST_TAGS=
+    set TEST_FSHARPQA_SUITE=0
+    set TEST_FSHARP_SUITE=0
     set TEST_VS=1
+    set TEST_TAGS=
 )
 
 if /i '%ARG%' == 'ci_part2' (
     set BUILD_PROTO=1
     set SKIP_EXPENSIVE_TESTS=1
     set BUILD_CORECLR=1
     set BUILD_PORTABLE=1
+
+    set TEST_COMPILERUNIT=0
+    set TEST_NET40_COREUNIT=0
+    set TEST_PORTABLE_COREUNIT=0
     set TEST_CORECLR=1
-    set TEST_FSHARPQA_SUITE=1
+    set TEST_FSHARPQA_SUITE=0
     set TEST_FSHARP_SUITE=1
+    set TEST_VS=0
+    set TEST_TAGS=
+)
+
+if /i '%ARG%' == 'ci_part3' (
+    set BUILD_PROTO=1
+    set SKIP_EXPENSIVE_TESTS=1
+    set BUILD_CORECLR=1
```

Comment:
Why are we building CORECLR without testing it? shouldn't it be already covered in ci_part2?